index.html

```diff
@@ -29,25 +29,112 @@
             </ul>
         </div>
     </header>
-    <main class="landing">
-        <div class="about-us">
-            <div class="social-icon">
-                <ul>
-                    <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
-                    <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
-                    <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
-                    <li><a href="#"><i class="fab fa-youtube "></i></a></li>
-                    <li><a href="#"><i class="fab fa-medium "></i></a></li>
+    <main>
+        <section class="landing">
+            <div class="about-us">
+                <div class="social-icon">
+                    <ul>
+                        <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
+                        <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
+                        <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
+                        <li><a href="#"><i class="fab fa-youtube "></i></a></li>
+                        <li><a href="#"><i class="fab fa-medium "></i></a></li>
+                    </ul>
+                </div>
+                <div class="intro ">
+                    <h1>Hey there. <br>I am Sediqullah Badakhsh</h1>
+                    <h2>I am a Software Developer</h2>
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+            </div>
+            <div>
+                <img class="landing-image " src="image/landig-image.jpg" alt="landing-image "> </div>
+        </section>
+        <section class="recent-works">
+            <h3>My Recent Works</h3>
+            <hr>
+            <div class="recent-work">
+                <a href=""><img src="./image/Img_Placeholder_1.png" alt="Women doing yoga"></a>
+                <h4>Multi-Post Stories</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tech-used">
+                    <li>CSS</li>
+                    <li>html</li>
+                    <li>bootsrap</li>
+                    <li>Ruby</li>
                 </ul>
+                <button class="project-button " type="button">See Project</button>
             </div>
```

Comment:
- [ ]  Please consider making your design here to match that of your chosen design. I suggest you align the content of each card with flexbox as I pointed out in another review so that you can properly align the contents properly so as to match your template and also to have a professional-looking page 😄 

Your design: 
<img width="219" alt="Loom_8QMQHaTnfD" src="https://user-images.githubusercontent.com/81468675/148625142-a91eaa91-28ce-4ef2-8707-eb1e7742b468.png">

Figma design:
<img width="208" alt="Loom_wC4ztX2FQV" src="https://user-images.githubusercontent.com/81468675/148625151-db4cc1dc-ecd5-4f0e-8952-16a58cdc7be7.png">